code/game/mecha/working/clarke.dm

```diff
@@ -0,0 +1,37 @@
+/obj/mecha/working/clarke
+	desc = "Combining man and machine for a better, stronger engineer. Can even resist lava!"
+	name = "\improper Clarke"
+	icon_state = "clarke"
+	max_temperature = 65000
+	max_integrity = 250
+	step_in = 1.5
+	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	lights_power = 7
+	deflect_chance = 10
+	step_energy_drain = 15 //slightly higher energy drain since you movin those wheels FAST
+	armor = list("melee" = 20, "bullet" = 10, "laser" = 30, "energy" = 30, "bomb" = 60, "bio" = 0, "rad" = 70, "fire" = 100, "acid" = 100) //low bullet/melee armor to compensate for fire protection and speed
```

Comment:
why 30 laser armor thats gygax tier of armor

Reply:
What are you on, Tlal? FF has the same laser armor.

Reply:
ff doesnt have wheels and op said "light armor high speed"